docs/getting-started-guides/ubuntu/troubleshooting.md

```diff
@@ -68,41 +68,11 @@ This will automatically ssh you to the easyrsa unit.
 
 ## Collecting debug information
 
-Sometimes it is useful to collect all the information from a node to share with a developer so problems can be identifying. This section will deal on how to use the debug action to collect this information. The debug action is only supported on `kubernetes-worker` nodes.
+Sometimes it is useful to collect all the information from a cluster to share with a developer so problems can be identified. This section will deal on how to use the [CDK Field Agent](https://github.com/juju-solutions/cdk-field-agent) to collect this information.
```

Comment:
This language could be simplified and made a bit more direct:

> Sometimes it is useful to collect all the information from a cluster to share with a developer so problems can be identified

could be:
Sometimes it is useful to collect all the information from a cluster to share with a developer to identify problems.

The next sentence doesn't quite read correctly, recommend:
> This section will deal on how to use the [CDK Field Agent](https://github.com/juju-solutions/cdk-field-agent) to collect this information.

to:

> This section reviews how to use the [CDK Field Agent](https://github.com/juju-solutions/cdk-field-agent) to collect this information.

Reply:
Completely agree here. I'll do this.